core/ui/public/i18n/en/translation.json

```
@@ -481,19 +482,24 @@
     "invalid_url": "Invalid URL",
     "backup_page_tooltip_1": "Here you can configure your backups and restore any app.",
     "backup_page_tooltip_2": "Cluster data is composed of <strong>cluster configuration data</strong> and <strong>apps data</strong>.",
-    "backup_page_tooltip_3": "Ensure your peace of mind by configuring:",
-    "backup_page_tooltip_4": "<strong>Cluster configuration backup</strong>: download it regularly and store it in a safe place",
-    "backup_page_tooltip_5": "<strong>Apps backup</strong>: add your backup repositories and configure apps backup scheduling",
-    "backup_page_tooltip_6": "You can restore any previously backed up app or the whole cluster:",
-    "backup_page_tooltip_7": "<strong>Restore an app</strong>: you can do it in the <strong>Restore</strong> section",
-    "backup_page_tooltip_8": "<strong>Restore the whole cluster</strong>: you need a <strong>cluster configuration backup</strong> and a fresh installation of {productName}. This is useful in a disaster recovery scenario",
-    "download_cluster_configuration_backup": "Download backup",
+    "apps_configuration_tooltip": "Add your backup repositories and configure apps backup scheduling",
+    "restore_tooltip": "You can restore any previously backed up app or the whole cluster:",
+    "backup_page_tooltip_8": "<strong>Diaster recovery</strong>: you will need a cluster configuration backup and a fresh installation of {productName}. The disaster recovery can be achieved only from the cluster configuration wizard.",
+    "download_cluster_configuration_backup": "Download cluster backup",
     "restore_app": "Restore an app",
     "replace_existing_app": "Replace existing app {app}",
     "cluster_configuration": "Cluster configuration",
-    "apps": "Apps",
+    "apps": "Applications",
```

```suggestion
    "apps": "Apps",
```
We use the term "app" everywhere else, maybe it's better to be consistent

---

I really do not like it, but ok ...